BootloaderCommonPkg/Library/SmbiosInitLib/SmbiosInitLib.c

```diff
@@ -244,7 +244,15 @@ GetSmbiosString (
 
   for (Index = 0; Index < PcdGet16 (PcdSmbiosStringsCnt); ++Index) {
     if (Type == SmbiosStrings[Index].Type && StringId == SmbiosStrings[Index].Idx) {
-      return SmbiosStrings[Index].String;
+      if (SmbiosStrings[Index].String != NULL) {
```

Comment:
@stalamudupula There are multiple "unknown" in this function. Can you do more clean-up?